cfgmgr/teammgr.cpp

```diff
@@ -395,6 +395,21 @@ bool TeamMgr::setLagMtu(const string &alias, const string &mtu)
     return true;
 }
 
+bool TeamMgr::setLacpKey(const string& alias, const string& lacp_key)
```

Comment:
This new function is not called anywhere ?

Reply:
Fixed